spec/requests/request_spec.rb

```diff
@@ -88,8 +88,8 @@
 
   describe 'pul_holdings check' do
     it 'excludes scsb items when pul location filter is applied' do
-      get '/catalog.json?per_page=100'
-      expect(response.body).to include('"SCSB-7235709"', '"9741216"')
+      # get '/catalog.json?per_page=150'
+      # expect(response.body).to include('"SCSB-7235709"', '"9741216"')
```

Comment:
Is this test change related?  It would be good to know why it should be removed, or keep it if there's no particular reason to remove it.

Reply:
I suggested to have it removed because now there are over 100 fixtures in the development environment. 100 is the max one can fetch at once in Blacklight. The test was failing because id SCSB-7235709 was on the 2nd page of the results.

Reply:
👍 that makes sense.  Can we delete it instead of commenting it out?